### PR TITLE
#16570 Show a warning when no task types are available; introduce `standalone` task type flag

### LINKED ISSUE #16570: Can't create tasks without any connections in the project
#### Describe the problem you're observing:
Tasks can't be created on an empty project without any connections:

image

This is a partially expected behavior since all<sup>1</sup> tasks operate over database objects, but it's not clear what's wrong in this case.

If there are no applicable task types we should display a message describing what's happening.

_<sup>1</sup> The `Shell command` task does not require a connection._

#### Steps to reproduce, if exist:

1. Switch to an empty project
2. Attempt to create a task

### PR DESCRIPTION
**Pull request by ShadelessFox**
java_Mm11UopLgD

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.tasks.ui/src/org/jkiss/dbeaver/tasks/ui/internal/TaskUIMessages.properties [hunk 1]
```diff
@@ -31,6 +31,7 @@ task_config_wizard_page_task_text_label_name = Name
 task_config_wizard_page_task_control_label_descr = Description
 task_config_wizard_page_task_text_label_task_id = Task ID
 task_config_wizard_page_task_create_folder_label = Task folder
+task_config_wizard_page_task_no_task_types_available = No suitable task types found for the current project.\nTo continue, please create a new connection.
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.tasks.ui/src/org/jkiss/dbeaver/tasks/ui/internal/TaskUIMessages.properties]:
```suggestion
task_config_wizard_page_task_no_task_types_available = No suitable task types found for the current project.\nTo continue please create a new connection.
```

**Review by LonwoLonwo — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by Matvey16** [comment]
Create task button disappeared
Expected:
dbeaver_34tVdMjwY0
Actual:
dbeaver_GLsGucsHfK

**Comment by LonwoLonwo** [comment]
@Matvey16 your case is here: https://github.com/dbeaver/dbeaver/issues/17904

**Comment by Matvey16** [comment]
@LonwoLonwo 
@ShadelessFox 
No, I compared it to other builds and this bug is specific to this branch, where those buttons don't appear no matter what. And closing DBeaver and installing any other build immidiately fixes it

dbeaver_5ZR2Ezwc3d

**Comment by ShadelessFox** [comment]
Perhaps this branch is just outdated. I have pulled changes from `devel`, please try again

**Comment by Matvey16** [comment]
Still the same

**Comment by ShadelessFox** [comment]
It's weird. I'm 100% sure that it's not caused by this PR as the changes only affect the task wizard. Can we just assume that everything is ok?

**Comment by Matvey16** [comment]
Verified

**Pull request merged**